package.json

```diff
@@ -19,6 +19,7 @@
     "version": "node bin/bump-dependencies.js && npm install && git add package.json package-lock.json",
     "postversion": "git push && git push --tags",
     "test": "mocha -t 10000 --recursive test",
+    "test:w": "mocha -t 10000 --recursive test --watch",
```

Comment:
This is a convenience I liked when developing. Can remove it. I found it a pain to have linting pass when I was working on making my tests pass.